main/FatesInterfaceMod.F90

```diff
@@ -1944,7 +1973,69 @@ subroutine UpdateFatesRMeansTStep(sites,bc_in)
            call cpatch%tveg24%UpdateRMean(bc_in(s)%t_veg_pa(ifp))
            call cpatch%tveg_lpa%UpdateRMean(bc_in(s)%t_veg_pa(ifp))
            call cpatch%tveg_longterm%UpdateRMean(bc_in(s)%t_veg_pa(ifp))
+           
+           ! Update seedling layer running means of PAR, soil matric potential and 
+           ! moisture deficit days (mdd).
+           n_leaf = maxval(cpatch%ncan(cpatch%ncl_p,:)) ! Calculate the number of leaf layers
+                                                        ! in the lowest canopy layer
+           
+           ! Calculate the fraction of total lai (summed across pfts) in sun vs. shade
+           ! at the lowest leaf level of the lowest canopy level           
+           lai_sun_frac = sum(cpatch%ed_laisun_z(cpatch%ncl_p,:,n_leaf)) &
+            / ( sum(cpatch%ed_laisun_z(cpatch%ncl_p,:,n_leaf)) + &   !summed across pfts  
+                sum(cpatch%ed_laisha_z(cpatch%ncl_p,:,n_leaf)) )    !summed across pfts
+           
+           lai_shade_frac = 1.0_r8 - lai_sun_frac     
+           
+           ! Calculate new_seedling_layer_par (total PAR at the lowest leaf layer of the lowest canopy layer)
+           ! using the weighted average of direct and diffuse par.
+           
+           ! Notes to code reviewers on calculating "new_seedling_layer_par" (4-11-2023):
+           ! 1. Charlie or Ryan, can you please verify that weighted-avg approach is indeed getting
+           ! *total* par at the lowest leaf layer of the lowest canopy layer?
+           ! 2. This variable currently might create a seedling layer par that is too dark because
+           ! it assumes that all seedlings are hiding under larger trees? Where the canopy is closed this is
+           ! OK, but in open forest it will overesimtae shade expected by seedlings I think?
+           ! What if the sun_frac weight above were multiplied by bc_in(s)%solad_parb(ifp,ipar) instead?
```

Comment:
I would say that the most critical thing here is to make sure that the light level for the seedling layer should be consistent with the absorbed light the cohorts would get in the photosynthesis scheme. If the layer par turns out to be too dark, but it is consistent with the canopy radiative transfer code, then I would keep it as is...

Reply:
Ok, the seedling layer par is consistent with the radiative transfer code because its the same light experienced by the lowest leaf layer in the understory. I think using this is OK because the amount of direct light received by the seedling layer is sensitive to LAI.